js/client.js

```diff
@@ -1228,7 +1228,7 @@
 				if (this.className === 'closebutton') return; // handled elsewhere
 				if (this.className.indexOf('minilogo') >= 0) return; // handled elsewhere
 				if (!this.href) return; // should never happen
-				if (this.host === 'play.pokemonshowdown.com' || this.host === 'replay.pokemonshowdown.com' || this.host === 'psim.us' || this.host === location.host) {
+				if ((this.host === 'play.pokemonshowdown.com' || this.host === 'replay.pokemonshowdown.com' || this.host === 'psim.us' || this.host === location.host) && this.className !== 'no-panel-intercept') {
```

Comment:
```suggestion
				if ([
					'play.pokemonshowdown.com', 'replay.pokemonshowdown.com', 'psim.us', location.host,
				].includes(this.host) && this.className !== 'no-panel-intercept') {
```

Line length is a bit long; edit for readability.